src/app/pages/vm/devices/device-add2/device-add.component.ts

```diff
@@ -200,7 +200,7 @@ export class DeviceAddComponent implements OnInit {
       type : 'select',
       name : 'sectorsize',
       placeholder : 'Disk sector size',
-      tooltip : 'Enter a sector size in bytes. <i>0</i> leaves the\
+      tooltip : 'Select a sector size in bytes. <i>0</i> leaves the\
```

Comment:
'0 leaves the sector size unset', but the label for '0' option is Default. Could you make them consistent?

Reply:
ok will make changes right now